frontend/src/forms/ParcelDetailForm.test.tsx

```diff
@@ -176,7 +176,7 @@ describe('ParcelDetailForm', () => {
       });
       const errors = getAllByText('Required');
       const idErrors = getAllByText('pid or pin Required');
-      expect(errors).toHaveLength(9);
+      expect(errors).toHaveLength(8);
```

Comment:
hmm, is there one less required field now?

Reply:
Right, have to add this option to the AutoComplete component. On it now, not going to merge yet